### PR TITLE
Add check on client-side to see if KnitServer exists before destroying it

### PR DESCRIPTION
**Pull request by aku-e**
I was testing a module that uses utility modules from Knit using the command bar on the client-side when Knit threw an error saying "KnitServer" does not exist.

I realized this was because the Knit module had already been required by client-side scripts, thus had deleted the KnitServer module, but since the command bar runs in a separate scope/context (not sure of the correct terminology here) it requires it again, then tries to find an instance that is no longer under the Knit module and errors.

This would allow for modules to be tested via the command bar during the run-time on the client-side.

### CODE REVIEW

**Comment by Sleitnick** [comment]
I wonder if there is a better way to go about testing modules than through the command line? I don't have any issues with this change, but it does highlight a possible need for a tool or something to help test

File: src/init.lua [hunk 1]
```diff
@@ -1,6 +1,9 @@
+local KnitServer = script:FindFirstChild("KnitServer")
 if (game:GetService("RunService"):IsServer()) then
-	return require(script.KnitServer)
+	return require(KnitServer)
 else
-	script.KnitServer:Destroy()
+	if KnitServer then
```
Comment (Sleitnick) [inline src/init.lua:5]: To conform to our styles, can you wrap this in parenths? e.g. `if (KnitServer) then`

**Review by Sleitnick — CHANGES_REQUESTED** [review]

File: src/init.lua (hunk 1 above)
Comment (aku-e) [reply src/init.lua]: Done 🙂

**Comment by aku-e** [comment]
I sometimes test code from the command bar as sometimes it's more convenient than writing some temporary code in some file somewhere, just to make sure a module works properly before starting work on a gameplay feature.

**Comment by aku-e** [comment]
> I sometimes test code from the command bar as sometimes it's more convenient than writing some temporary code in some file somewhere, just to make sure a module works properly before starting work on a gameplay feature.

Oops, forgot to mention: yes, there probably is a better way of going about testing modules.

**Comment by aku-e** [comment]
My bad didn't know that I need to re-request a review once I've implemented the changes the review wants. I'm still pretty new to GitHub pull request stuff😅

**Review by Sleitnick — APPROVED** [review]

**Pull request merged**